prime-router/build.gradle.kts

```diff
@@ -39,7 +39,7 @@ plugins {
     id("jacoco")
     id("org.jetbrains.dokka") version "1.7.10"
     id("com.avast.gradle.docker-compose") version "0.16.9"
-    id("org.jetbrains.kotlin.plugin.serialization") version "1.6.21"
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.7.20"
```

Comment:
I don't think this plugin is being used. @greene858 can you verify this?  This was added in https://github.com/CDCgov/prime-reportstream/pull/6374

Reply:
@carlosfelix2 we do use this. It's the extension for the compiler that helps us build our token serialization for REST and API senders.